requirements.txt

```diff
@@ -0,0 +1,2 @@
+jsonschema
+future
```

Comment:
Is `future` missing for some version and need to be pip installed? Or does `past` come from it and is missing in some built in versions?